wgs/config/config.py

```diff
@@ -140,7 +140,6 @@ def default_params(mode='all'):
 
 def refdir_data(refdir):
     yamlpath = os.path.join(refdir, 'metadata.yaml')
-
```

Comment:
why?

Reply:
why did i delete the line? sorry, i really have trouble understanding github comments for some reason.

Reply:
yeah, 

did you remove to make it more compliant with pep-8? is there a reason?